openwisp_monitoring/device/api/views.py

```diff
@@ -200,6 +201,38 @@ def _write(self, request, pk):
                 metric.write(client['mac'])
             if created:
                 self._create_clients_graph(metric)
+        if 'resources' in data:
+            resources = {}
+            extra_values = {}
+            if 'load' in data['resources']:
+                extra_values['load'] = {
+                    'load_1': 100 * data['resources']['load'][0],
+                    'load_15': 100 * data['resources']['load'][2],
+                }
+                resources['load'] = 100 * data['resources']['load'][1]
```

Comment:
Please do the following changes:

Save the key as load_1, and for extra values use load_5 and load_15. Remember to not multiply *100. We will have float Thresholds soon :+1: